### PR TITLE
OLS-1121: Doc about system prompt

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Doc about system prompt

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [ ] CI configuration change
- [ ] Konflux configuration change

## Related Tickets & Documents

- Related Issue #[OLS-1121](https://issues.redhat.com//browse/OLS-1121)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -431,6 +431,14 @@ Depends on configuration, but usually it is not needed to generate or use API ke
 ## 12. Registering new LLM provider
```
Comment (JoaoFula) [inline README.md:431]: Registering **a** new LLM provider (unrelated I know, but I saw it)

File: README.md [hunk 2]
```diff
@@ -431,6 +431,14 @@ Depends on configuration, but usually it is not needed to generate or use API ke
...
    Please look [here](https://github.com/openshift/lightspeed-service/blob/main/CONTRIBUTING.md#adding-a-new-providermodel) for more info.
 
+## 13. Fine tuning
+   Service uses so called system prompt to put the question into context before the question is send to selected LLM. The default system prompt is fine tuned for questions about OpenShift and Kubernetes. It is possible to use different system prompt via the configuration option `system_prompt_path` in section `ols_config`. That option must contain path to text file with actual system prompt (can contain multiple lines). An example of such configuration:
```
Comment (JoaoFula) [inline README.md:435]: The service uses **the,** so called**,** system prompt to...

Comment (JoaoFula) [reply README.md:435]: ... before the question is **sent** to **the** selected LLM

Comment (JoaoFula) [reply README.md:435]: It is possible to use **a** different system prompt

Comment (JoaoFula) [reply README.md:435]: ... in **the** `ols_config` section

Comment (JoaoFula) [reply README.md:435]: That option must contain **the** path to **the** text file

File: README.md (hunk 1 above)
Comment (tisnik) [reply README.md]: fixed, thank you

File: README.md (hunk 2 above)
Comment (tisnik) [reply README.md]: fixed, thank you

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Pull request merged**